python/flatbuffers/util.py

```diff
@@ -20,6 +20,14 @@ def GetSizePrefix(buf, offset):
 	"""Extract the size prefix from a buffer."""
 	return encode.Get(packer.int32, buf, offset)
 
+def GetBufferIdentifier(buf, offset, size_prefixed = False):
+        """Extract the file_identifier from a buffer"""
+        offset+=(number_types.UOffsetTFlags.bytewidth if size_prefixed else 0) + number_types.UOffsetTFlags.bytewidth # offset sizeof(root table offset) + sizeof(size prefix) if present   
+        return buf[offset:offset+encode.FILE_IDENTIFIER_LENGTH]
+
+def BufferHasIdentifier(buf, offset, file_identifier, size_prefixed = False):
+        return GetBufferIdentifier(buf, offset, size_prefixed = size_prefixed)==file_identifier
```

Comment:
Please break this out into 2-3 lines by assigning the values to intermediate variables.

Reply:
@rw did you mean

     offset+=(number_types.UOffsetTFlags.bytewidth if size_prefixed else 0) + number_types.UOffsetTFlags.bytewidth

Reply:
@pyottamus yes!